app/views/partials/theme/simple/comments/edit.liquid

```diff
@@ -0,0 +1,5 @@
+<div class="container flex flex-wrap items-center mx-auto">
+  <h1 class="text-2xl">{{ 'app.models.comments.edit' | t }} {{ object.name  }}</h1>
```

Comment:
This is the place where I see that titles in a form of a partial are not a good idea and I'll replace it with class name soon.